app/assets/javascripts/components/students/student_drawer.jsx

```diff
@@ -24,6 +25,10 @@ const StudentDrawer = React.createClass({
     };
   },
 
+  componentWillUnmount() {
```

Comment:
Could this be done from StudentList instead of the drawer? There are lots of drawers mounted (even if just one is open), but just one StudentList, and closeDrawer doesn't depend on knowing which drawer is open.